app/controllers/definitions_controller.rb

```diff
@@ -0,0 +1,14 @@
+class DefinitionsController < ApplicationController
+	def create 
+		@definition = Definition.new(
+			definition_params
+		)
+		@definition.entry_id = params[:entry_id]
```

Comment:
where does this param get set? does rails do this for you?

Reply:
its defined on line 11. But it does seem less readable this way. This I'll fix

Reply:
entry_id is set by rails using foreign keys. When I set the belongs_to and has_many relationships, rails gives us the functionality for free